geopandas/array.py

```diff
@@ -56,7 +57,7 @@ def construct_array_type(cls):
 register_extension_dtype(GeometryDtype)
 
 
-def _check_crs(left, right, allow_none=False):
+def _check_crs(left, right, allow_none: bool = False) -> bool:
```

Comment:
why not `left` and `right` add typing hint?

Reply:
Where I didn't add type hints was when it wasn't clear what all types were allowed as an argument for the parameter.